Editorial review dev product guide (#140)
ansys/DevRelDocs: Markdown/developer_product_guide/Digital_Mission_Engineering_DME.md

# Digital Mission Engineering (DME)

DME is part of Ansys Government Initiatives (AGI).

* [Ansys Government Initiatives (AGI) GitHub](https://github.com/AnalyticalGraphicsInc)

## Systems Tool Kit (STK)

> * [Digital Mission Engieering Products: 2024 R1](https://www.agi.com/new-stk)

Reviewer: There may be a better solution but I'm not sure what

> * [Digital Mission Engieering Products: 2024 R1](https://www.agi.com/new-stk)

Author: ```suggestion
* [Digital Mission Engieering Products](https://www.agi.com/new-stk)
```

* [STK Python Programmer's Guide](https://help.agi.com/stkdevkit/index.htm#python/pythonProgrammingGuide.htm?TocPath=Development%2520Environments%257CSTK%2520Python%2520API%257CProgrammer%25E2%2580%2599s%2520Guide%257C_____0)
* [STK Java API](https://help.agi.com/stkdevkit/index.htm#java/jdkOverview.htm?TocPath=Development%2520Environments%257CSTK%2520Java%2520API%257C_____0)

### Ansys STK training material

* [Ansys AGI Training and Certification (AGI)](https://www.agi.com/Training-and-Certification)
* [Python and SssssssTK! (AGI)](https://www.agi.com/tech-tips/Python-and-SssssssTK)
* [Lesson 16-2 Integrating STK With Python Using Jupyter Notebooks (AGI)](https://www.youtube.com/watch?v=uaVVzpRDg3s)

> ## DME component libraries

Author: ```suggestion
## DME Component Libraries
```

> You canse Ansys DME component libraries with Python through [Pythonnet](https://pythonnet.github.io/) or with MATLAB.

Author: ```suggestion
You can use Ansys DME Component Libraries with Python through [Pythonnet](https://pythonnet.github.io/) or with MATLAB.
```

> ## Ansys DME component libraries training material

Author: I see that you are correct about it being a proper noun. It is used on our website as a product name: Ansys DME Component Libraries.
